### DISCUSSION TITLE
Issues with Kratos Selfservice UI React Native project

### DISCUSSION THREAD

**Discussion by mrvinceit** [ory/.github#32] [Q&A]
When running the latest [Self-Service UI for React Native](https://github.com/ory/kratos-selfservice-ui-react-native/commit/55a212f5b15b35dfc713be59ff49da9820e63dd2), I'm getting `Error: Request failed with status code 308` this is happening with both the playground project and Ory Cloud projects.

Any thoughts on how to resolve?

**Comment by aeneasr** [comment]
Are you using [this guide](https://www.ory.sh/react-native-authentication-login-signup/) to run the example?

**Reply by mrvinceit** [reply]
Well yes, largely because the example does not suggest any additional customization and it defaults to the playground environment: https://github.com/ory/kratos-selfservice-ui-react-native/blob/6a59a5606316611129f88390943e20108d7dbe57/app.config.js#L9.

**Reply by mrvinceit** [reply]
Is there any hope to get this working? As anyone tested this against the latest build of Ory Console? I'm really stuck here, in particular, I'm trying to work out getting social providers for Ory and have been having several different issues which are out of scope for this issue, but it is very concerning that the Selfservice react native project does not work out of the box even though it should. 

As an aside, I evaluated Ory and this project about 8 months ago and I believed things were working, I no longer have the version of the project so it's not much help in this case, but just thought I'd mention it.

**Reply by aeneasr** [reply]
Hi, I can't look into this today and tomorrow, but will be able to take a look on friday. Can you please share the steps you took like:

1. git clone
2. npm i
3. export FOO_BAR=...
4. npm run start
5. ...

Thanks!

**Reply by mrvinceit** [reply]
Greatly appreciated!!

Steps taken:

1. git clone git@github.com:ory/kratos-selfservice-ui-react-native.git
2. npm i
3. npm run android

Here is a screenshot of signup -- notice that we are connecting to the *playground* environment:
![image](https://user-images.githubusercontent.com/62821770/200976854-8e0a6242-5844-4930-bd59-e7417100147a.png)

```javascript
Network request failed, Object {
  "config": Object {
    "adapter": [Function xhrAdapter],
    "data": "{\"csrf_token\":\"\",\"traits.email\":\"test@example.com\",\"password\":\"password1234!\",\"method\":\"password\"}",
    "headers": Object {
      "Accept": "application/json, text/plain, */*",
      "Content-Type": "application/json",
    },
    "maxBodyLength": -1,
    "maxContentLength": -1,
    "method": "post",
    "timeout": 10000,
    "transformRequest": Array [
      [Function transformRequest],
    ],
    "transformResponse": Array [
      [Function transformResponse],
    ],
      "silentJSONParsing": true,
    },
    "url": "https://playground.projects.oryapis.com/api/kratos/public/self-service/registration?flow=21acafe8-6117-4502-9445-22d52f8c3481",
    "validateStatus": [Function validateStatus],
    "withCredentials": false,
    "xsrfCookieName": "XSRF-TOKEN",
    "xsrfHeaderName": "X-XSRF-TOKEN",
  },
  "error": [Error: Request failed with status code 308],
}
```

**Reply by aeneasr** [reply]
Ah yes, try changing

```diff
-   KRATOS_URL = "https://playground.projects.oryapis.com/api/kratos/public",
+   KRATOS_URL = "https://playground.projects.oryapis.com",
```

that should do the trick
